README.md

```diff
@@ -1418,6 +1418,35 @@ Other Style Guides
     }
     ```
 
+    <a name="variables--unary-increment-decrement"></a><a name="13.5"></a>
+    - [13.5](#variables--unary-increment-decrement) Avoid using unary increments and decrements (++, --). Eslint [`no-plusplus`](http://eslint.org/docs/rules/no-plusplus)
+
+      > Why? Per the Eslint documentation, unary increment and decrement statements are subject to automatic semicolon insertion and can cause silent errors with incrementing or decrementing values within an application.  It is also more expressive to mutate your values with statements like `num += 1` instead of `num ++`.  Disallowing unary increment and decrement statements also prevents you from pre-incrementing/pre-decrementing values unintentionally which can also cause unexpected behavior in your programs.
+
+      ```javascript
+        // bad
+
+        let array = [1, 2, 3];
+        let num = 1;
+        let increment = num ++;
+        let decrement = -- num;
+
+        for(let i = 0; i < array.length; i++){
+          array[i];
+        }
+
+        // good
+
+        let array = [1, 2, 3];
+        let num = 1;
+        let increment = num += 1;
+        let decrement = num -= 1;
+
+        for(let i = 0; i < array.length; i += 1){
```

Comment:
in this case, the guide discourages ever using loops, so the "good" example should be
```
array.forEach((item) => {
  doSomethingWithItem(item);
});
```